File: 5. Redémarrer.go
package main

import (
	"fmt"
	"os/exec"
)

func main() {
	service := "Spooler"
	fmt.Println("Redémarrage du service", service)
	exec.Command("cmd", "/C", "net stop "+service).Run()
	exec.Command("cmd", "/C", "net start "+service).Run()
	fmt.Println("Service redémarré")
}

File: 5. Redémarrer.py
import os

service_name = "Spooler"  # exemple : Print Spooler, change selon tes tests

print(f"Redémarrage du service {service_name}...")
os.system(f'net stop "{service_name}"')
os.system(f'net start "{service_name}"')
print("Service redémarré.")
